jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -202,6 +202,16 @@ public synchronized ResponseEntity<String> depositTim(String jsonString, Request
       String obfuscatedTimData = TimTransmogrifier.obfuscateRsuPassword(odeTimData.toJson());
       stringMsgProducer.send(odeProperties.getKafkaTopicOdeTimBroadcastJson(), null, obfuscatedTimData);
 
+      // If the TIM was generated by the TMC broadcast to the filtered TMC TIM Kafka topic
+      try {
+         String generatedBy = (new JSONObject(obfuscatedTimData)).getJSONObject("metadata").getString("recordGeneratedBy");
```

Comment:
The TIMs coming into this endpoint will always be marked as TMC as this is the entry point for them to be generated. I'd like to discuss with the team on this one where it might make sense to pull the unique TMC messages from. If the system will be generating TIM messages solely through the ODE, it may make sense to only pull them from here and not in teh Asn1DecodeDataRouter. I think a good offline discussion will help us hash this one out a bit

Reply:
I've moved this message deposit from the Asn1DecodedDataRouter and the TimDepositController to the Asn1EncodedDataRouter after the phase 1 message encoding.